src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,46 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_NUMBER_OF_ELEMENTS = 10;
+    private K[] keysArray;
+    private V[] valuesArray;
+    private int currentSize;
```

Comment:
let's fix naming, it will be enough to understand
```suggestion
    private K[] keys;
    private V[] values;
    private int size;
```